src/Documentation/Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -134,6 +134,9 @@ A quick way to test this is to (see in the aforementioned the few commented line
 The ADO.NET persistence has functionality to version data and define arbitrary (de)serializers with arbitrary application rules and streaming, but currently
 there is no method to expose them to application code. More information in [ADO.NET Persistence Rationale](#ADONETPersistenceRationale).
 
+__NB for Microsoft SQL Server users:__
+For the ADO.NET Storage Provider to work with SQL Server, you need to add [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) NuGet package to your project.
```

Comment:
Doesn't a similar thing apply for MySQL and other supported SQL provider implementations?